libs/types/src/investments.rs

```diff
@@ -185,7 +185,7 @@ pub struct Swap<
 	pub currency_in: Currency,
 	/// The outgoing currency, i.e. the one which should be replaced.
 	pub currency_out: Currency,
-	/// The amount of outgoing currency which shall be exchanged.
+	/// The amount of incoming currency which shall be bought.
```

Comment:
See [this comment](https://github.com/centrifuge/centrifuge-chain/pull/1509#discussion_r1331346309) for explanation.